extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -55,6 +56,12 @@ export class BaseSlides extends BaseCarousel {
     this.shouldAutoplay_ = this.hasAutoplay_ && this.isLoopingEligible();
 
     if (this.shouldAutoplay_) {
+      this.registerAction('toggleAutoPlay', invocation => {
```

Comment:
Currently you toggle it on/off using the same function and by passing in a bool - Happy to change that to two separate functions if you find that more ergonomic.